pkg/keyspace/keyspace.go

```diff
@@ -96,6 +97,8 @@ type Manager struct {
 	kgm *GroupManager
 	// nextPatrolStartID is the next start id of keyspace assignment patrol.
 	nextPatrolStartID uint32
+	// cached keyspace name for each keyspace ID.
+	keyspaceNameLookup sync.Map // store as ID(uint32) -> name(string)
```

Comment:
I don't see keyspace name deleted or updated. Is that expected?

Reply:
Currently, we do not have logic for modifying ID and name or deleting keyspace.